test_libs/pyspec/eth2spec/test/sanity/test_blocks.py

```diff
@@ -312,35 +332,35 @@ def test_voluntary_exit(spec, state):
 # @with_all_phases
 # @spec_state_test
 # def test_transfer(spec, state):
-    # overwrite default 0 to test
-    # spec.MAX_TRANSFERS = 1
+# overwrite default 0 to test
```

Comment:
why reduce comment depth here?

Reply:
similar as with `test_eth1_data_votes_...`, wanted to re-enable commented tests with modifications. But forgot to keep the old indentation in place when reverting on this one.

Reply:
will rebase it out